src/sentry/api/issue_search.py

```diff
@@ -92,6 +94,30 @@ def convert_status_value(value, projects, user, environments):
     return parsed
 
 
+def convert_category_value(value, projects, user, environments):
```

Comment:
Hey what does the converter do if the feature flag is off? looks like it returns None right now yeah?